setup.py

```diff
@@ -20,9 +20,8 @@
 ]
 
 testing_extras = [
-    'coverage',
-    'nose',
-    'nosexcover',
+    'pytest',
+    'pytest-cov',
```

Comment:
a) testing tools are usually just needed for developers and not installed for users of a software.
developers use virtualenv and pip, so they don't need the debian stable package, even if they use debian stable.
b) until letsencrypt is finished, the current debian testing might become the new stable and it has pytest-cov. testing is already frozen IIRC.